skills/csharp/experimental/itsmskill/Dialogs/Teams/TicketTaskModule/CreateTicketTeamsImplementation.cs

```diff
@@ -0,0 +1,138 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace ITSMSkill.Dialogs.Teams.TicketTaskModule
+{
+    using System;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using AdaptiveCards;
+    using ITSMSkill.Dialogs.Teams.CreateTicketTaskModuleView;
+    using ITSMSkill.Extensions.Teams;
+    using ITSMSkill.Models;
+    using ITSMSkill.Services;
+    using ITSMSkill.TeamsChannels;
+    using ITSMSkill.TeamsChannels.Invoke;
+    using Microsoft.Bot.Builder;
+    using Microsoft.Bot.Schema.Teams;
+    using Microsoft.Extensions.DependencyInjection;
+    using Newtonsoft.Json.Linq;
+    using Attachment = Microsoft.Bot.Schema.Attachment;
+
+    /// <summary>
+    /// CreateTicketTeamsImplementation Handles OnFetch and OnSumbit Activity for TaskModules
+    /// </summary>
+    [TeamsInvoke(FlowType = nameof(TeamsFlowType.CreateTicket_Form))]
+    public class CreateTicketTeamsImplementation : ITeamsTaskModuleHandler<TaskModuleResponse>
+    {
+        private readonly IStatePropertyAccessor<SkillState> _stateAccessor;
+        private readonly ConversationState _conversationState;
+        private readonly BotSettings _settings;
+        private readonly BotServices _services;
+        private readonly IServiceManager _serviceManager;
+
+        public CreateTicketTeamsImplementation(
+             IServiceProvider serviceProvider)
+        {
+            _conversationState = serviceProvider.GetService<ConversationState>();
+            _settings = serviceProvider.GetService<BotSettings>();
+            _services = serviceProvider.GetService<BotServices>();
+            _stateAccessor = _conversationState.CreateProperty<SkillState>(nameof(SkillState));
+            _serviceManager = serviceProvider.GetService<IServiceManager>();
+        }
+
+        // Handle Fetch
+        public async Task<TaskModuleResponse> OnTeamsTaskModuleFetchAsync(ITurnContext context, CancellationToken cancellationToken)
+        {
+            return new TaskModuleResponse()
+            {
+                Task = new TaskModuleContinueResponse()
+                {
+                    Value = new TaskModuleTaskInfo()
+                    {
+                        Title = "ImpactTracker",
+                        Height = "medium",
```

Comment:
"medium" seems to be used in multiple places. define it as a constant and reuse it

Reply:
added TaskModuleConstants.Medium and TaskModuleConstants.Continue